source/admin_manual/list_of_events.rst

```diff
@@ -1136,6 +1184,61 @@ sessions, and/or detect broken clients.
 +---------------------+------------------------------------------------------+
 
 
+IMAP Hibernate
+==============
+
+.. versionadded:: v2.3.13
+
++---------------------+------------------------------------------------------+
+| Field               | Description                                          |
++=====================+======================================================+
+| user                | Username of the user                                 |
++---------------------+------------------------------------------------------+
+| session             | Session ID of the IMAP connection                    |
++---------------------+------------------------------------------------------+
+| mailbox             | Mailbox name where hibernation was started in.       |
++---------------------+------------------------------------------------------+
+| local_ip            | IMAP connection's local (server) IP                  |
++---------------------+------------------------------------------------------+
+| local_port          | IMAP connection's local (server) port                |
++---------------------+------------------------------------------------------+
+| remote_ip           | IMAP connection's remote (client) IP                 |
++---------------------+------------------------------------------------------+
+| remote_port         | IMAP connection's remote (client) port               |
++---------------------+------------------------------------------------------+
+
+.. _imap_hibernate_client_unhibernated:
```

Comment:
Isn't this wrong?

Reply:
There are two "imap_client_unhibernated" events in different processes, so there needs to be two labels. So there is already a "_imap_client_unhibernated" label and this needs to be named something else. I just happened to choose this name.